ethportal-peertest/src/utils.rs

```diff
@@ -138,20 +141,7 @@ fn read_state_fixture_from_file(file_name: &str) -> Result<Vec<StateFixture>> {
     let yaml_content = fs::read_to_string(file_name)?;
     let value: Value = serde_yaml::from_str(&yaml_content)?;
 
-    let mut result = vec![];
-
-    for fixture in value.as_sequence().unwrap() {
-        result.push(StateFixture {
-            content_data: StateContentData::deserialize(fixture)?,
-            recursive_gossip: fixture.get("recursive_gossip").and_then(|value| {
-                if value.is_null() {
-                    None
-                } else {
-                    Some(StateContentData::deserialize(value).unwrap())
-                }
-            }),
-        });
-    }
+    let result = <Vec<StateFixture>>::deserialize(value)?;
```

Comment:
```suggestion
    let result = <Vec<StateFixture>>::deserialize(value)?;
```
Do we need the extra <>, maybe we do it is just my first time seeing something like this :monocle_face: